pyx/mathutils.py

```diff
@@ -109,7 +109,7 @@ def _realroots_quartic(a3, a2, a1, a0):
 
 
 def realpolyroots(*cs):
-    """returns the roots of a polynom with given coefficients
+    r"""returns the roots of a polynom with given coefficients
```

Comment:
This seems to be an unnecessary raw string.

Reply:
Sorry, my fault. All fine.!

Reply:
Python 3.8 is complaining about the `\s` in `\sum` in the fourth line of the docstring. Therefore, it seems that we need to keep it.